snippets/0901_StopwatchVsDateTime/index.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: default
+codename: StopwatchVsDateTime
+title: Stopwatch és DateTime használata időmérésre
+tags: time, snippet
+authors: Faragó Timea
+---
+
+# Stopwatch és DateTime használata időmérésre
+Hasznos tud lenni ha már valamilyen meglévő kódot tesztelgetünk, elemezgetünk például  gyorsaság szempontból. Vannak helyzetek, amikor a helyes működés nem elegendő és szeretnénk megtudni, hogy mi lassít az alkalmazásunkon és szükségünk van egy pontos időmérésre alkalmas eszközre. Most két ilyen lehetőségről lesz só, melyek közül az egyik...
+## DateTime.Now
```

Comment:
Hogy a plain text is olvasható legyen, nem lenne jobb a fejlécek elé egy üres sor?